programs/github

```diff
@@ -144,10 +147,4 @@ if action == 'auth' then
 	return
 end
 
-if not validActions[action] then
-	for k, v in pairs(validActions) do
-		print(v.usage)
-	end
-else
-	print(validActions[action].usage)
-end
+print(action.usage)
```

Comment:
Does this handle the case where we fail to find a valid action? Or is that handled above?